index.js

```diff
@@ -154,46 +165,45 @@ const recastNewUsers = async () => {
     if (USERNAMES_TO_SKIP.includes(user.username)) {
       continue
     }
-    if (recastCounts[user.address] >= MAX_RECAST_PER_USER) {
+    if (recastCounts[user.fid] >= MAX_RECAST_PER_USER) {
       console.log(
         `Skipping ${user.username} because casted more than ${MAX_RECAST_PER_USER} times already.`
       )
       continue // Skip because exceeded max recasts
     }
-    const casts = await getCasts(user.address)
-    let recastCount = recastCounts[user.address]
-    for (const cast of casts.result.casts.reverse()) {
+    const casts = await getCasts(NCBOT_FID)
+    let recastCount = recastCounts[user.fid]
+    for (const cast of casts.result.casts) {
       if (recastCount > MAX_RECAST_PER_USER) {
         console.log(
           `Skipping the rest of ${user.username}'s casts since recasted ${MAX_RECAST_PER_USER} times already.`
         )
         break
       }
 
-      const { body, meta } = cast
-      const { address, data, publishedAt, sequence, username } = body
-      if (meta.recast) {
+      const { hash, timestamp, recasts, parentHash, text } = cast
+      if (recasts?.count) {
         continue // Skip because recasts
       }
-      if (publishedAt < publishedAtThreshold) {
+      if (timestamp < publishedAtThreshold) {
         continue // Skip if casted before the threshold
       }
-      if (data.replyParentMerkleRoot) {
+      if (parentHash) {
         continue // Skip replies
       }
       if (
-        latestSequences[address] !== undefined &&
-        sequence <= latestSequences[address]
+        latestSequences[username] !== undefined &&
+        sequence <= latestSequences[username]
```

Comment:
is `username` always equivalent to `fid`?

Reply:
just fixed this, it should be fid!